srcpkgs/meson/patches/musl-tests.patch

```diff
@@ -0,0 +1,79 @@
+# see https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/main/meson/skip-broken-tests.patch
+# more specifically https://github.com/mesonbuild/meson/issues/8283
+# previously there was a vsed in the post_patch() of the template
+--- a/mesonbuild/utils/universal.py
++++ b/mesonbuild/utils/universal.py
+@@ -19,6 +19,7 @@
+ from functools import lru_cache, wraps
+ from itertools import tee
+ from tempfile import TemporaryDirectory, NamedTemporaryFile
++from packaging.tags import sys_tags
+ import typing as T
+ import textwrap
+ import pickle
+@@ -115,6 +116,7 @@
+     'is_aix',
+     'is_android',
+     'is_ascii_string',
++    'is_musl',
+     'is_cygwin',
+     'is_debianlike',
+     'is_dragonflybsd',
+@@ -650,6 +652,10 @@
+ def is_wsl() -> bool:
+     return is_linux() and 'microsoft' in platform.release().lower()
+
++def is_musl() -> bool:
++    tags = list(sys_tags() )
++    return "musl" in tags[0].platform
```

Comment:
If you want this upstream it needs to do something like [check for `__MUSL__`](https://wiki.musl-libc.org/faq#Q:-Why-is-there-no-%3Ccode%3E__MUSL__%3C/code%3E-macro?).

Don't use third-party libraries (meson has none by design and this adds a runtime dep solely to get a testsuite hack to work) which depend on `import sys; analyze_elffile(sys.executable)` to check if DT_INTERP contains the word "musl" and fails when python is statically linked or `CC=musl-gcc` + a glibc system.